### DISCUSSION TITLE
Jobs are missing while adding to the queue

### DISCUSSION THREAD

**Discussion by aravindbalaji-ngp** [taskforcesh/bullmq#1718] [Q&A]
I have a scenario where i will be adding 100 jobs to the queue but the queue takes only 97 or 96 (I have cross verified the total using a counter), it is hit or miss can anyone suggest any solutions to this.

**Comment by manast** [comment]
Post a minimal test code that exhibit this behaviour and we will look into it.

**Reply by aravindbalaji-ngp** [reply]
I optimized my code and everything works fine now thanks for the reply.
